packages/calcite-components/src/components/flow-item/flow-item.tsx

```diff
@@ -389,8 +391,10 @@ export class FlowItem
             <slot name={SLOTS.fab} slot={PANEL_SLOTS.fab} />
             <slot name={SLOTS.contentTop} slot={PANEL_SLOTS.contentTop} />
             <slot name={SLOTS.contentBottom} slot={PANEL_SLOTS.contentBottom} />
-            <slot name={SLOTS.footerActions} slot={PANEL_SLOTS.footerActions} />
             <slot name={SLOTS.footer} slot={PANEL_SLOTS.footer} />
+            <slot name={SLOTS.footerActions} slot={PANEL_SLOTS.footerActions} />
+            <slot name={SLOTS.footerEnd} slot={PANEL_SLOTS.footerEnd} />
```

Comment:
I originally thought all footer slots would render regardless of slotted content, but this will slot into `panel`, which has the mutually exclusive `footer` and `footer-start`/`footer-end` slot rendering, right? I like this!

Reply:
Brilliant! ⭒°｡𖥔 ݁ ˖𖦹⭒°｡⋆⭒

Reply:
The `flow-item` was failing to render `footer-start` and `footer-end`, I figured it was because slotting an empty slot in this structure gives the empty footer precedence. I had to do a little refactoring to make it work. 
<img width="466" alt="Screenshot 2024-06-24 at 2 09 16 PM" src="https://github.com/Esri/calcite-design-system/assets/19231036/d778cf9d-e527-49aa-bab5-8f41adc9e070">

https://github.com/Esri/calcite-design-system/pull/9374/files#diff-36d423bf7b7d6ed67ae5d1bd668b38396b501071d0cda69791e514f68bfd5b94R395-R398